jupyterlab_h5web/utils.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from typing import Union
+
+
+def _resolve_path(file_path: Path, base_dir: Path) -> Path:
+    if file_path.is_absolute():
+        return file_path
+
+    return base_dir / file_path
+
+
+def resolve_file_path(
+    file_path: Union[str, Path], base_dir: Union[str, Path]
```

Comment:
Inverting both parameters would make it easier to read since in the end it does `base_dir/file_path`

Reply:
or eventually a docstring.

Reply:
and/or another name making it explicit the returned path is absolute (e.g., `as_absolute_path`)